docs/docs-site/content/developer/components/_index.md

```diff
@@ -31,7 +31,7 @@ To run the [demo app](https://github.com/cloudera/hue/tree/master/tools/examples
 2. In [package.json](https://github.com/cloudera/hue/blob/master/tools/examples/api/hue_dep/package.json), remove `"hue": "file:../../../.."` without touching the newly added gethue dependency
 3. In [src/app.js](https://github.com/cloudera/hue/blob/master/tools/examples/api/hue_dep/src/app.js), change the import line to:
 
-       import sqlAutocompleteParser from 'gethue/parse/sql/hive/hiveAutocompleteParser';
+       import sqlAutocompleteParser from 'gethue/lib/parsers/hiveAutocompleteParser';
```

Comment:
Bueno!

Might break current ER in 'Relationships' tab in Table browser, to check after